Data/Sys/GameSettings/GED.ini

```diff
@@ -2,6 +2,7 @@
 
 [Core]
 # Values set here will override the main Dolphin settings.
+CPUThread = 0
```

Comment:
I could add that, but it seems like we don't have any game INIs that set fake-completion yet... @JMC47, is there a specific reason for that, or have there just not been any INIs that we wanted to set fake-completion in?